packages/shared/__tests__/ReactSymbols-test.internal.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+'use strict';
+
+describe('ReactSymbols', () => {
+  beforeEach(() => jest.resetModules());
+
+  const expectToBeUnique = keyValuePairs => {
+    const map = new Map();
+    keyValuePairs.forEach(([key, value]) => {
+      if (map.has(value)) {
+        throw Error(
+          `${key} value ${value.toString()} is the same as ${map.get(value)}.`,
+        );
+      }
+      map.set(value, key);
+    });
+  };
+
+  it('Symbol values should be unique', () => {
+    expectToBeUnique(Object.entries(require('shared/ReactSymbols')));
+  });
+
+  it('numeric values should be unique', () => {
+    global.Symbol.for = null;
```

Comment:
I think you'll want to restore this after the test has run. It's a global so I'd expect it to be shared between tests. At least within one file so the order starts mattering.

```
const originalSymbolFor = global.Symbol.for;
global.Symbol.for = null;
try {
  // ...
} finally {
  global.Symbol.for = originalSymbolFor;
}
```

Reply:
I thought Jest automatically reset modules between test files. 🤔

But yeah, still probably a good suggestion.

Reply:
Agreed 👍